### ISSUE TITLE
Check File Descriptors useage should not break on 503s

### ISSUE THREAD

**Issue by majormoses** [sensu-plugins/sensu-plugins-elasticsearch#41]
When getting a 503 it should rescue:
```
Check failed to run: 503 Service Unavailable, ["/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:74:in `return!'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:495:in `process_result'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit'", "/opt/sensu/embedded/lib/ruby/2.2.0/net/http.rb:853:in `start'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/sensu-plugins-elasticsearch-0.3.0/bin/check-es-file-descriptors.rb:87:in `get_es_resource'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/sensu-plugins-elasticsearch-0.3.0/bin/check-es-file-descriptors.rb:95:in `acquire_es_version'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/sensu-plugins-elasticsearch-0.3.0/bin/check-es-file-descriptors.rb:110:in `acquire_max_fds'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/sensu-plugins-elasticsearch-0.3.0/bin/check-es-file-descriptors.rb:125:in `run'", "/opt/sensu/embedded/lib/ruby/gems/2.2.0/gems/sensu-plugin-1.2.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"]
```

I am not sure how we should handle it, I am leaning towards making it an unknown rather than a critical as you probably have other checks that are more suited to see if its up. Before implementing I think I should see what others think.

**Comment by majormoses** [comment]
I was using 0.3.0 I am testing now with latest and am not seeing the issue although I see no changes to error handling maybe my test is not good enough I will upgrade and see if I can replicate in one of our environments.

**Comment by majormoses** [comment]
created #43 to fix. I am open to other ideas.

**Comment by majormoses** [comment]
closed via: #43
